acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/OptimisticSafeSlotsSyncPostMergeAcceptanceTest.java

```diff
@@ -0,0 +1,115 @@
+  private static final String NETWORK_NAME = "less-swift";
+  private static final Integer SAFE_SLOTS_TO_IMPORT_OPTIMISTICALLY = 4;
+
+  private final SystemTimeProvider timeProvider = new SystemTimeProvider();
+  private BesuNode eth1Node1;
+  private BesuNode eth1Node2;
+  private TekuNode tekuNode1;
+  private TekuNode tekuNode2;
+  private String eth1Node1Enode;
+
+  @BeforeEach
+  void setup() throws Exception {
+    final int genesisTime = timeProvider.getTimeInSeconds().plus(10).intValue();
+    eth1Node1 =
+        createBesuNode(
+            config ->
+                config
+                    .withMiningEnabled()
+                    .withMergeSupport(true)
+                    .withP2pEnabled()
+                    .withGenesisFile("besu/preMergeGenesis.json"));
+    eth1Node1.start();
+    eth1Node1Enode = eth1Node1.fetchEnodeUrl();
+    eth1Node2 =
+        createBesuNode(
+            config ->
+                config
+                    .withMergeSupport(true)
+                    .withP2pEnabled()
+                    .withGenesisFile("besu/preMergeGenesis.json")
+                    .withStaticNodes(eth1Node1Enode));
+    eth1Node2.start();
+
+    final int totalValidators = 4;
+    final ValidatorKeystores validatorKeystores =
+        createTekuDepositSender(NETWORK_NAME).sendValidatorDeposits(eth1Node1, totalValidators);
+    tekuNode1 =
+        createTekuNode(
+            config ->
+                configureTekuNode(config, eth1Node1, genesisTime)
+                    .withValidatorKeystores(validatorKeystores)
+                    .withValidatorProposerDefaultFeeRecipient(
+                        "0xFE3B557E8Fb62b89F4916B721be55cEb828dBd73"));
+    tekuNode1.start();
+    tekuNode2 =
+        createTekuNode(
+            config ->
+                configureTekuNode(config, eth1Node2, genesisTime)
+                    .withInteropValidators(0, 0)
+                    .withPeers(tekuNode1)
+                    .withSafeSlotsToImportOptimistically(SAFE_SLOTS_TO_IMPORT_OPTIMISTICALLY));
+    tekuNode2.start();
+  }
+
+  @Test
+  void shouldPassMergeOptimisticallyAndBeginFinalizationAfterSafeSlotsToImport() throws Exception {
+    tekuNode2.waitForGenesis();
+    // Disconnect eth1Node2 from eth1Node1 before the merge
+    assertThat(eth1Node2.removePeer(eth1Node1Enode)).isTrue();
+    tekuNode1.waitForLogMessageContaining("MERGE is completed");
+
+    tekuNode2.waitForLogMessageContaining("optimistic sync");
+    tekuNode2.waitForLogMessageContaining("MERGE is completed");
+    tekuNode2.waitForNonDefaultExecutionPayload();
+    tekuNode2.waitForNewFinalization();
+    Optional<GetBlockRootResponse> getBlockRootResponse2 = tekuNode2.fetchBeaconRootHead();
+    assertThat(getBlockRootResponse2.get().execution_optimistic).isTrue();
+    Optional<GetBlockRootResponse> getBlockRootResponse1 = tekuNode1.fetchBeaconRootHead();
+    assertThat(getBlockRootResponse1.get().execution_optimistic).isFalse();
```

Comment:
We shouldn't be exposing `fetchBeaconRootHead` as a getter style here - it should be a wait method like `waitForHeadExecutionOptimistic` so that the DSL automatically waits.  Otherwise it's too easy to introduce intermittency into tests by not waiting for the right thing (or because later the thing to wait for changes and we don't find all the places in tests to update).